compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -28,7 +28,7 @@ use crate::ty::fast_reject::SimplifiedType;
 use crate::ty::util::Discr;
 pub use adt::*;
 pub use assoc::*;
-pub use generic_args::*;
+pub use generic_args::{GenericArgKind, *};
```

Comment:
? is that not just the following or does the explicit import overwrite something
```suggestion
pub use generic_args::*;
```

Reply:
It means that the type alias is imported with higher precedence than the glob that imports GenericArg from type_ir crate